### ISSUE TITLE
 watchVector option

### ISSUE THREAD

**Issue by luciman** [mapgears/ol3-google-maps#56]
In `olgm.OLGoogleMaps`one can set the `watchVector` option.
The default is set to `true`, but if you set it to `false` and the google layer is visible the vector layers are not included in google map and the opacity of vector layers becomes 0.
I think that for the case where `watchVector` is `false` the opacity of vector layers should remain 1 or that of the initial opacity.

This (watchVector false and opacity 1) can be a temporary solution for #35

**Comment by adube** [comment]
Hello.

I'm not sure I fully understand.  Would you please create a JSFiddle that demonstrates the issue ?

**Comment by luciman** [comment]
Sorry, now when I tried to create a simple example I saw everything is ok.
It was a mistake from me, because my map has two instances of olgm.OLGoogleMaps, one for the main map and one for the overview map and I set watchVector: false just for the main one. The vector layers get opacity 0 from the overview olgm, because both of them use the same layers.
Thanks for your promptitude and sorry for the noise.
